kong/plugins/myplugin/schema.lua

```diff
@@ -1,7 +1,7 @@
-local typedefs = require "kong.db.schema.typedefs"
+local PLUGIN_NAME = "myplugin"
```

Comment:
`plugin_name` should lowercase for the schema definition to pull it in. Also I think putting the require back at the top make things a little more readable (up to you on that one, not strong feeling either way).

```suggestion
local typedefs = require "kong.db.schema.typedefs"

local plugin_name = "myplugin"
```